source/catalog.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="header">
+    <div class="header-logo"><img src="https://via.placeholder.com/150x150" alt=""></div>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="nav-wrapper">
+        <ul class="nav__list site-list">
```

Comment:
здесь не понятно откуда взялся элемент .nav__list (блока .nav не нахожу). возможно так?!
```suggestion
        <ul class="main-nav__list site-list">
```
и если используются миксины классов по БЭМ, лучше писать сначала класс блока, а затем класс элемента. так легче искать блоки среди классов
```suggestion
        <ul class="site-list main-nav__list">
```